### PR TITLE
added simple baselayout

### PR DESCRIPTION
**Pull request by verosh**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -40,3 +40,4 @@ testem.log
 # System files
 .DS_Store
 Thumbs.db
+angular.json
```
Comment (lestherll) [inline .gitignore]: Remove [angular.json](angular.json) from [.gitignore](.gitignore)

File: src/app/toolbar/toolbar.component.html
Comment (lestherll) [inline src/app/toolbar/toolbar.component.html:1]: We will decompose this into different components at some point but should be fine for now.

**Review by lestherll — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (verosh) [reply .gitignore]: done

File: src/app/toolbar/toolbar.component.html
Comment (verosh) [reply src/app/toolbar/toolbar.component.html:1]: yea its placeholders currently

**Review by lestherll — APPROVED** [review]

**Pull request merged**